src/appkit.cs

```diff
@@ -10551,6 +10694,65 @@ interface NSImageView : NSAccessibilityImage, NSMenuItemValidation {
 		[Mac (11, 0)]
 		[Export ("symbolConfiguration", ArgumentSemantic.Copy)]
 		NSImageSymbolConfiguration SymbolConfiguration { get; set; }
+
+		// from the category NSSymbolEffect (NSImageView)
+
+		[Mac (14, 0)]
+		[Export ("addSymbolEffect:")]
+		void AddSymbolEffect (NSSymbolEffect symbolEffect);
+
+		[Mac (14, 0)]
+		[Export ("addSymbolEffect:options:")]
+		void AddSymbolEffect (NSSymbolEffect symbolEffect, NSSymbolEffectOptions options);
+
+		[Mac (14, 0)]
+		[Export ("addSymbolEffect:options:animated:")]
+		void AddSymbolEffect (NSSymbolEffect symbolEffect, NSSymbolEffectOptions options, bool animated);
+
+		[Mac (14, 0)]
+		[Export ("removeSymbolEffectOfType:")]
+		void RemoveSymbolEffect (NSSymbolEffect symbolEffect);
+
+		[Mac (14, 0)]
+		[Export ("removeSymbolEffectOfType:options:")]
+		void RemoveSymbolEffect (NSSymbolEffect symbolEffect, NSSymbolEffectOptions options);
+
+		[Mac (14, 0)]
+		[Export ("removeSymbolEffectOfType:options:animated:")]
+		void RemoveSymbolEffect (NSSymbolEffect symbolEffect, NSSymbolEffectOptions options, bool animated);
+
+		[NoiOS, Mac (14, 0)]
```

Comment:
I believe the NoiOS is not needed here?
```suggestion
		[Mac (14, 0)]
```